client/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:5.9
```

Comment:
It wasn't 27 days ago when I wrote this.  I can push it 5.10 but I don't think it's worth the effort right now